subsys/bluetooth/audio/tbs_client.c

```diff
@@ -1545,6 +1545,12 @@ static uint8_t primary_discover_gtbs(struct bt_conn *conn, const struct bt_gatt_
 	return BT_GATT_ITER_STOP;
 }
 
+static void initialize_net_buf_read_buffer(struct bt_tbs_instance *inst)
+{
+	net_buf_simple_init_with_data(&inst->net_buf, &inst->read_buf, sizeof(inst->read_buf));
```

Comment:
nitpick: It might be good to break the line here.

Reply:
Indeed. The (strongly) recommended maximum line length is still 80 characters.